string soma
string soma_config
string region_id 
time start_from
time until
duration duration 
